src/main/java/fi/aalto/cs/apluscourses/intellij/model/APlusProject.java

```diff
@@ -25,6 +25,11 @@ public Path getBasePath() {
     return Paths.get(Objects.requireNonNull(project.getBasePath()));
   }
 
+  @NotNull
+  public Path getCourseFilePath() {
+    return getBasePath().resolve(Paths.get(Project.DIRECTORY_STORE_FOLDER, "a-plus-project"));
```

Comment:
`FileUtilRt.toSystemIndependentName(...)` or will work wo?

Reply:
Java's `Path` class handles paths in a system independent way.